src/app/settings/components/settings-container.component.ts

```diff
@@ -35,8 +35,7 @@ export class SettingsContainerComponent implements OnInit, OnDestroy {
     { value: 'de', label: 'de' },
     { value: 'sk', label: 'sk' },
     { value: 'fr', label: 'fr' },
-    { value: 'es', label: 'es' },
-    { value: 'pt-br', label: 'pt-br' }
+    { value: 'es', label: 'es' }
```

Comment:
Could it be that this is a leftover from the rebase, or was it intentionally to remove `pt-br`?